topdown/http.go

```diff
@@ -20,13 +20,21 @@ import (
 	"strings"
 	"time"
 
+	"github.com/open-policy-agent/opa/topdown/cache"
```

Comment:
[nit] let's lump it together with the import block below

Reply:
Done